core/src/primitives/rectangle/mod.rs

```diff
@@ -268,24 +268,93 @@ impl Rectangle {
     /// );
     /// ```
     pub fn resized(&self, size: Size, anchor_point: AnchorPoint) -> Self {
+        let mut resized = self.clone();
+        resized.resize_width_mut(size.width, anchor_point.x());
+        resized.resize_height_mut(size.height, anchor_point.y());
+
+        resized
+    }
+
+    /// Returns a resized rectangle with the given width.
+    ///
+    /// The rectangle is resized relative to the given anchor edge.
```

Comment:
I'm not convinced this is better, but it's nice to have the full behaviour described in one line. Might be helpful for some IDEs that only show the first line too. If you think the current wording is better, feel free to leave it as is.

```suggestion
    /// Returns a new rectangle with the given width, resized relative to the given anchor edge.
```